android/build.gradle

```diff
@@ -108,20 +99,11 @@ android {
   }
 
   buildTypes {
-    debug {
```

Comment:
Why it was removed?

Reply:
Copied the latest build.gradle from builder.bob, I guess none of this options are no longer needed

Reply:
Without this we won't be able to debug the library with android studio debugger

Reply:
Let's add it back and I will merge the pr.

Reply:
ok